draft-kuehlewind-taps-crypto-sep.md

```diff
@@ -49,14 +49,15 @@ informative:
 
 --- abstract
 
-Due to the latency involved in connection setup and security handshakes, there is an increasing
-deployment of cryptographic session resumption mechanisms. While cryptographic context and
-endpoint capabilities need to be be known before encrypted application data can be sent, there is
-otherwise no technical constraint that the cryptographic handshake must be performed on the same
-transport connection. This document recommends a logical separation between the mechanism(s) used to negotiate
-capabilities and set up the initial encryption context as well as maintain this context (control protocol), 
-the application of encryption and
-authentication state to data (record protocol), and the associated transport connection(s).
+Secure transport protocols often consist of three logically distinct components: transport, control (handshake),
+and record protection. Protocols such as TLS couple these pieces for ease of implementation and specification. Others,
+such as QUIC, define a cleaner separation of concerns, wherein TLS is used as the control
+logic that negotiates secrets and QUIC uses those secrets for packet protection. This separation is
```

Comment:
It also feels a bit odd to say "TLS does X." "QUIC on the other hand does Y, and uses TLS this way." I know that's how it actually goes, but without more text here it raises more questions than it answers.